src/core/config/openthread-core-config-check.h

```diff
@@ -441,4 +441,9 @@
     "OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_METRICS_ENABLE was replaced by OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE."
 #endif
 
+#ifdef OPENTHREAD_CONFIG_MAC_MAX_FRAME_RETRIES_DIRECT
```

Comment:
Can we do the same for `OPENTHREAD_CONFIG_MAC_MAX_FRAME_RETRIES_INDIRECT` define?

Reply:
I forgot about that one. Thanks.